src/view/search/search-view.tsx

```diff
@@ -100,22 +99,23 @@ const SearchView: FC<SearchProps> = ({ useQuery, ResultsHeader }) => {
 					offset: reset ? 0 : searchResults.offset,
 					sortBy: searchResults.sortBy
 				})
-			) // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-				// @ts-ignore
-				.then(({ payload }: any) => {
+			)
+				.then(({ payload }) => {
 					setLoading(true);
```

Comment:
I think that _setLoading_ should be inside the conditional block

Reply:
I think it is not needed as it is already setted to true before at line 87. I don't know why would anyone set to true something already set to true. In this PR I am just making sure to perform operation which involv3es payload only if payload exists. I didn't change any other behaviour